docs/Blog-Post.md

```diff
@@ -0,0 +1,578 @@
+# ---
+layout: blog
+title: Levanter —  Legible, Scalable, Reproducible Foundation Models with JAX
+authors:
+    - name: David Hall
+      url: TODO
+    - name: Percy Liang
+      url: https://cs.stanford.edu/~pliang/
+display: False
+---
+> <div class="blog-tagline">
+    <strong> We introduce <a href="https://github.com/stanford-crfm/levanter" target="_blank">Levanter</a>, our codebase
+         for training reproducible, legible foundation models models using JAX. We also
+        release a number of checkpoints for models trained with Levanter, including new architectures, on our
+        <a href="https://huggingface.co/stanford-crfm" target="_blank">Hugging Face Hub</a> page.
+    </strong>
+> </div>
+
+
+# Introduction
+
+We are excited to announce the release of [Levanter](https://github.com/stanford-crfm/levanter), a new [JAX](https://github.com/google/jax)-based codebase for training foundation models.
+Levanter is designed to be legible, scalable, and reproducible:
+
+1. Legible: Levanter comes with a new named tensor library named Haliax that makes it easy to write legible, composable deep learning code, while still being high performance.
+2. Scalable: Levanter is designed to scale to large models, and to be able to train on a variety of hardware, including GPUs and TPUs.
+3. Reproducible: Levanter is bitwise deterministic, meaning that the same configuration will always produce the same results, even in the face of preemption and resumption.
+
+Today, we're releasing the first version of Levanter, along with [tutorials](https://colab.research.google.com/drive/1TiTcQQ4V5mopbgCu1SVl-oqJtXn7rFnC)
```

Comment:
Have explicit versioning?  Levanter 1.0?

Reply:
I'm always scared of 1.0, but ok